src/plugins/saved_objects_management/public/management_section/objects_table/components/flyout.tsx

```diff
@@ -855,7 +862,8 @@ export class Flyout extends Component<FlyoutProps, FlyoutState> {
     let confirmButton;
 
     let importButtonDisabled = false;
-    if (this.props.dataSourceEnabled && this.props.hideLocalCluster && !selectedDataSourceId) {
+    // If a data source is enabled, the import button should be disabled when there's no selected data source
+    if (this.props.dataSourceEnabled && selectedDataSourceId === undefined) {
```

Comment:
Do we need to check for local cluster? If local cluster is a option here, should we let the import button to be available?

Reply:
if localCluster is an option, `selectedDataSourceId`  will be an empty string, and it won't get into this condition, and import button will be available.

Reply:
Basically enable/disable import button shouldn't be correlated with `hideLocalCluster`, it's related to if there's any options being selected. If no selected option we should just disable the import

Reply:
Got it.